src/EditorFeatures/CSharpTest/InvertLogical/InvertLogicalTests.cs

```diff
@@ -0,0 +1,159 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.CSharp.InvertLogical;
+using Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.CodeRefactorings;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.InvertLogical
+{
+    public partial class InvertLogicalTests : AbstractCSharpCodeActionTest
```

Comment:
>InvertLogicalTests [](start = 25, length = 18)

Could you add a test showing the refactoring doesn't trigger on some other operators (like `|`)?

Reply:
sure.